Reference/Searching/Examine/index.md

```diff
@@ -22,3 +25,5 @@ A detailed overview from top to bottom of how to use Examine
 Shows all configuration options with an explanation for each
 
 ## [API - Examine Manager](examine-manager.md)
+
+Describes the singleton object which exposes all of the index and search providers which are registered in the configuration of the Umbraco application.
```

Comment:
Not sure about this one: Should we mention the singleton common pitfalls here?
Help @Jeavon @marcemarc

Reply:
Maybe it's worth providing a link to the Common Pitfalls page that mentions being weary with the singletons on the `examine-manager.md` page?

Reply:
Cool, added it to the Examine Manager page intro to check the pitfalls page.